src/components/Question.tsx

```diff
@@ -0,0 +1,350 @@
+      background-color: ${colors.palette.white};
+      background-clip: padding-box;
+      border-radius: 0.3rem;
+      max-width: ${layouts.answer.feedback.popover.maxWidth};
+      margin: calc(${layouts.answer.feedback.arrow.height} - 5px) 0 ${layouts.answer.horizontalSpacing} calc(-1 * (2 * ${layouts.answer.horizontalSpacing}));
+      box-shadow: 10px 0px 10px rgba(0, 0, 0, .25);
+      color: ${colors.palette.neutral};
+      font-size: 1.4rem;
+      font-style: italic;
+
+      .arrow {
+        position: absolute;
+        display: block;
+        width: ${layouts.answer.feedback.arrow.width};
+        height: ${layouts.answer.feedback.arrow.height};
+        margin-left: 30px;
+        top: calc((${layouts.answer.feedback.arrow.height} + ${layouts.answer.feedback.popover.borderWidth}) * -1);
+
+        &::before,
+        &::after {
+          position: absolute;
+          display: block;
+          content: "";
+          border-color: transparent;
+          border-style: solid;
+          border-width: 0 calc(${layouts.answer.feedback.arrow.width} / 2) ${layouts.answer.feedback.arrow.height} calc(${layouts.answer.feedback.arrow.width} / 2);
+        }
+        &::before {
+          top: 0;
+          border-bottom-color: ${colors.answer.feedback.popover.borderColor};
+        }
+        &::after {
+          top: ${layouts.answer.feedback.popover.borderWidth};
+          border-bottom-color: ${colors.palette.white};
+        }
+      }
+
+      .question-feedback-content {
+        padding: ${layouts.answer.feedback.popover.verticalSpacing} ${layouts.answer.feedback.popover.horizontalSpacing};
+      }
+    }
+  }
+
+  .openstax-answer {
+    border-top: 1px solid #d5d5d5;
+    margin: 10px 0;
+    padding: 10px 0;
+  }
+}
+`;
+
+export interface QuestionProps {
+  question: QuestionType,
+  answer_id?: string;
+  task?: {
+    is_deleted: boolean;
+    type?: 'homework'
+  } | null,
+  correct_answer_id: ID;
+  incorrectAnswerId: ID;
+  hideAnswers: boolean;
+  hidePreambles: boolean,
+  exercise_uid?: string;
+  displayFormats: boolean,
+  processHtmlAndMath: boolean;
+  className: string;
+  questionNumber: number;
+  displaySolution: boolean;
+  context?: string;
+  correct_answer_feedback_html?: string;
+  contentRenderer?: JSX.Element;
+  feedback_html: string;
+  onChange: () => void;
+  show_all_feedback?: boolean;
+  children?: ReactNode;
+}
+
+export const Question = (props: QuestionProps) => {
+  let exerciseUid, solution;
+
+  const {
+    question, correct_answer_id, incorrectAnswerId, exercise_uid, className, questionNumber, context, task, hidePreambles
+  } = props;
+
+  const { stem_html, collaborator_solutions, formats, stimulus_html } = question;
+
+  const hasCorrectAnswer = !!correct_answer_id;
+  const hasIncorrectAnswer = !!incorrectAnswerId;
+
+  const taskIsDeleted = (task != null ? task.is_deleted : undefined)
+  const taskIsHomework = ((task != null ? task.type : undefined) === 'homework');
```

Comment:
these feel like tutor cruft but maybe I just don't understand how they'll be used